src/ORM/Association/BelongsTo.php

```diff
@@ -55,6 +55,20 @@ public function getForeignKey(): array|string|false
         return $this->_foreignKey;
     }
 
+    /**
+     * Sets the name of the field representing the foreign key to the target table.
```

Comment:
```suggestion
     * Sets the field name for the foreign key in the source table.
```

Reply:
Copy pasta is going to be my death one day. Would you like me to update the wording in the parent too (including the getter)?

Reply:
The documentation could use some love. I'm only commenting because I'm reading it. If you don't mind updating the relevant comments, that would be nice. If not, I'll remember to review the wording on this sometime.

Reply:
Sorry, wasn't quick enough. I'll do a follow up if you don't beat me to it.

Reply:
It's good enough. We can touch up the docs all at once at some later date.